components/gallery/GalleryItem.vue

```diff
@@ -6,19 +6,23 @@
       :subtitle="$t('mint.successCreateNewNft', [congratsNewNft])" />
     <div class="columns is-variable is-6">
       <div class="column is-two-fifths">
-        <div class="is-relative">
+        <div id="nft-img-container" ref="imgref" class="is-relative">
```

Comment:
we don't really use ids in the code base
is it necessary here?
prefer class

Reply:
I need to use the priority of the id selector to override the style when it's not fullscreen